tsa-core/src/main/kotlin/org/usvm/machine/interpreter/TvmCellInterpreter.kt

```diff
@@ -460,11 +461,66 @@ class TvmCellInterpreter(
             is TvmCellParseCdepthInst -> visitDepthInst(scope, stmt, operandType = TvmCellType)
             is TvmCellParseSdepthInst -> visitDepthInst(scope, stmt, operandType = TvmSliceType)
             is TvmCellParseLdrefrtosInst -> visitLoadRefRtosInst(scope, stmt)
+            is TvmCellParseSdsubstrInst -> visitSdsubstrInst(scope, stmt)
 
             else -> TODO("Unknown stmt: $stmt")
         }
     }
 
+    fun visitSdsubstrInst(scope: TvmStepScopeManager, stmt: TvmCellParseSdsubstrInst) = with(ctx) {
+        scope.consumeDefaultGas(stmt)
+
+        val sizeBits = scope.takeLastIntOrThrowTypeError() ?: return
+        val offsetBits = scope.takeLastIntOrThrowTypeError() ?: return
```

Comment:
Since these are arbitrary integers, we must check that they are in slice index bounds. If not, an exception should be thrown (check concrete execution to see which one). Also, it would be better to have tests on such cases